this is still experimental in rust-1.46 for arm and aarch64

--- gfx/qcms/src/lib.rs.orig	2020-11-17 22:03:09.868746869 +0100
+++ gfx/qcms/src/lib.rs	2020-11-17 22:03:27.795605120 +0100
@@ -47,8 +47,8 @@
 pub mod transform;
 #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
 pub mod transform_avx;
-#[cfg(any(target_arch = "aarch64", target_arch = "arm"))]
-pub mod transform_neon;
+//#[cfg(any(target_arch = "aarch64", target_arch = "arm"))]
+//pub mod transform_neon;
 #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
 pub mod transform_sse2;
 pub mod transform_util;
--- gfx/qcms/src/transform.rs.orig	2020-11-17 22:15:03.413104913 +0100
+++ gfx/qcms/src/transform.rs	2020-11-17 22:15:17.128996464 +0100
@@ -21,11 +21,11 @@
 // OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-#[cfg(any(target_arch = "arm", target_arch = "aarch64"))]
-use crate::transform_neon::{
-    qcms_transform_data_bgra_out_lut_neon, qcms_transform_data_rgb_out_lut_neon,
-    qcms_transform_data_rgba_out_lut_neon,
-};
+//#[cfg(any(target_arch = "arm", target_arch = "aarch64"))]
+//use crate::transform_neon::{
+//    qcms_transform_data_bgra_out_lut_neon, qcms_transform_data_rgb_out_lut_neon,
+//    qcms_transform_data_rgba_out_lut_neon,
+//};
 use crate::{
     chain::qcms_chain_transform,
     double_to_s15Fixed16Number,
@@ -1424,18 +1424,18 @@
                         (*transform).transform_fn = Some(qcms_transform_data_bgra_out_lut_sse2)
                     }
                 }
-            } else if cfg!(any(target_arch = "arm", target_arch = "aarch64")) && qcms_supports_neon
-            {
-                #[cfg(any(target_arch = "arm", target_arch = "aarch64"))]
-                {
-                    if in_type == QCMS_DATA_RGB_8 {
-                        (*transform).transform_fn = Some(qcms_transform_data_rgb_out_lut_neon)
-                    } else if in_type == QCMS_DATA_RGBA_8 {
-                        (*transform).transform_fn = Some(qcms_transform_data_rgba_out_lut_neon)
-                    } else if in_type == QCMS_DATA_BGRA_8 {
-                        (*transform).transform_fn = Some(qcms_transform_data_bgra_out_lut_neon)
-                    }
-                }
+           // } else if cfg!(any(target_arch = "arm", target_arch = "aarch64")) && qcms_supports_neon
+           // {
+           //     #[cfg(any(target_arch = "arm", target_arch = "aarch64"))]
+           //     {
+           //         if in_type == QCMS_DATA_RGB_8 {
+           //             (*transform).transform_fn = Some(qcms_transform_data_rgb_out_lut_neon)
+           //         } else if in_type == QCMS_DATA_RGBA_8 {
+           //             (*transform).transform_fn = Some(qcms_transform_data_rgba_out_lut_neon)
+           //         } else if in_type == QCMS_DATA_BGRA_8 {
+           //             (*transform).transform_fn = Some(qcms_transform_data_bgra_out_lut_neon)
+           //         }
+           //     }
             } else if in_type == QCMS_DATA_RGB_8 {
                 (*transform).transform_fn = Some(qcms_transform_data_rgb_out_lut_precache)
             } else if in_type == QCMS_DATA_RGBA_8 {
